package.json

```diff
@@ -10,7 +10,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@wormhole-foundation/wormhole-connect": "^1.1.0",
+    "@wormhole-foundation/wormhole-connect": "1.1.3-beta.0-development",
```

Comment:
Is it okay to use the development version here?

Reply:
yes, only the development version contains the hotfix at the moment